index.html

```
@@ -6582,92 +6595,136 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
-                  false).
+                <li>For each <var>paramNameInOption</var> → <var>value</var> of
+                <var>options</var>:
+                  <ol>
+                    <li>If there exists an entry with name member equal to
+                    <var>paramNameInOption</var> inside
+                    <var>audioParamMap</var>, set that <a>AudioParam</a>'s
+                    value to <var>value</var>.
+                    </li>
+                  </ol>
                 </li>
-                <li>Perform <a href=
-                "https://tc39.github.io/ecma262/#https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">
-                  Set</a>(O=<var>parameterMap</var>, P=<var>paramName</var>,
-                  V=<var>audioParam</var>).
+                <li>Set <var>node</var>'s <a href=
+                "#widl-AudioWorkletNode-parameters">parameters</a> to
+                <var>audioParamMap</var>.
                 </li>
               </ol>
             </li>
-            <li>Perform <a href=
-            "https://tc39.github.io/ecma262/#sec-createmethodproperty">CreateDataProperty</a>(O=<var>this</var>,
-            P="parameters", V=<var>parameterMap</var>).
+            <li>
+              <a href="#queuing">Queue a control message</a> to create an
+              <a>AudioWorkletProcessor</a>, given <var>nodeName</var>,
+              <var>processorPortSerialization</var>, and <var>node</var>.
+            </li>
+            <li>Return <var>node</var>.
+            </li>
+          </ol>
+          <p>
+            In order to process a control message for the construction of an
+            <a>AudioWorkletProcessor</a>, given a string <var>nodeName</var>, a
+            serialization record <var>processorPortSerialization</var>, and an
+            <a>AudioWorkletNode</a> <var>node</var>, perform the following
+            steps on the <a>rendering thread</a>. If any of these steps throws
```

According to @bfgeek, `WorkletGlobalScope` does not have any error handling. So any exception that is being thrown here will be silently ignored. I think this issue should be handled by the worklet infra.